src/blocks/blocks/tabs/group/edit.js

```diff
@@ -199,7 +198,7 @@ const Edit = ({
 
 	const addTab = () => {
 		const itemBlock = createBlock( 'themeisle-blocks/tabs-item', {
-			title: __( 'Tab ', 'otter-blocks' ) + ( ( children?.length ?? 0 ) + 1 )
+			title: __( 'Tab', 'otter-blocks' ) + ( ( children?.length ?? 0 ) + 1 )
```

Comment:
This will also be better served with sprintf. We want there to be a space in between.